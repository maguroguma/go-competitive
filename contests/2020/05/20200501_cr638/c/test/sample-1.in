6
4 2
baba
5 2
baacb
5 3
baacb
5 3
aaaaa
6 4
aaxxzz
7 1
phoenix
