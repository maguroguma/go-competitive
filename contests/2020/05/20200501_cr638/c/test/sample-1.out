ab
abbc
b
aa
x
ehinopx
